bundles/org.openhab.core.io.rest.sitemap/src/main/java/org/openhab/core/io/rest/sitemap/SitemapSubscriptionService.java

```diff
@@ -215,67 +217,88 @@ public boolean exists(String subscriptionId) {
      * @return the name of the current sitemap or null if no sitemap is currently set for the subscription
      */
     public @Nullable String getSitemapName(String subscriptionId) {
-        String sitemapWithPageId = pageOfSubscription.get(subscriptionId);
+        String sitemapWithPageId = scopeOfSubscription.get(subscriptionId);
         return (sitemapWithPageId == null) ? null : extractSitemapName(sitemapWithPageId);
     }
 
     private String extractSitemapName(String sitemapWithPageId) {
         return sitemapWithPageId.split(SITEMAP_PAGE_SEPARATOR)[0];
     }
 
+    private boolean isPageListener(String sitemapWithPageId) {
+        return sitemapWithPageId.contains(SITEMAP_PAGE_SEPARATOR);
+    }
+
     private String extractPageId(String sitemapWithPageId) {
         return sitemapWithPageId.split(SITEMAP_PAGE_SEPARATOR)[1];
     }
 
     /**
-     * Updates the subscription to send events for the provided page id.
+     * Updates the subscription to send events for the provided page id (or whole sitemap if page is is null).
```

Comment:
Typo: page **id** is null

Reply:
wrote pageId to use the same spelling as the parameter